packages/core/src/styles/_type.scss

```diff
@@ -0,0 +1,100 @@
+@import "../../node_modules/@carbon/themes/scss/themes";
+
+svg.chart-svg {
+	text {
+		font-size: 12px;
+		line-height: 16px;
+		stroke-width: 0.3;
+		@include fonts();
+		font-weight: 400;
+
+		&.title {
+			font-size: 18px;
+			font-weight: bold;
+		}
+
+		&.chart-label {
+			font-family: "IBM Plex Sans Condensed", "IBM Plex Sans", sans-serif;
+			dominant-baseline: middle;
+		}
+
+		&.chart-title {
+			font-size: 14px;
+			line-height: 16px;
+			font-weight: 600;
+			padding-bottom: 8px;
+			dominant-baseline: hanging;
+		}
+	}
+	g.tick {
+		text {
+			font-family: "IBM Plex Sans Condensed", "IBM Plex Sans", sans-serif;
+		}
+	}
+ }
+
```

Comment:
Does carbon have tokens that will let us simplify this?

Reply:
done!